hieradata/integration.yaml

```diff
@@ -64,6 +64,9 @@ govuk_crawler::targets:
 
 govuk_elasticsearch::dump::run_es_dump_hour: '9'
 
+govuk_search::gor::enabled: true
+govuk_search::gor::output_path: '/data/vhost/rummager.integration.publishing.service.gov.uk/gor_dump'
```

Comment:
I would set this to a plainer location (something like `/etc/gor/dumps`) because otherwise it depends on the app resource for rummager being deployed, which could cause dependency errors.

Reply:
removed as I can just use the default